components/title/basic/src/index.js

```diff
@@ -0,0 +1,51 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+/**
+ * Builds the modifier className to apply to the component following the SUIT naming convention (sui-TitleBasic--modifierInCamelCase).
+ * @param alignProp {string} property value for Title alignment.
+ * @returns {string} SUIT formatted classname.
+ */
+const _buildAlignClassName = (alignProp) => {
```

Comment:
I know it does not exist the "private" concept in js, and although I am against this as well as you, I have seen that other similar functions in other sui-components includes the underscore dangle at the beggining of the functions names, so I will maintain this naming convention.